assets/src/edit-story/components/library/mediaLibrary.js

```diff
@@ -168,29 +168,30 @@ function MediaLibrary({ onInsert }) {
    *
    * @param {Object} evt Doc Event
    */
-  const onSearch = (evt) => {
-    setSearchTerm(evt.target.value);
-    setIsMediaLoading(false);
-    setIsMediaLoaded(false);
-  };
+  const onSearch = useCallback(
+    (evt) => {
+      setSearchTerm(evt.target.value);
+      reloadMedia();
+    },
+    [reloadMedia, setSearchTerm]
+  );
 
   /**
    * Filter REST API calls and re-request API.
    *
    * @param {string} filter Value that is passed to rest api to filter.
    */
-  const onFilter = (filter) => {
-    if (filter !== mediaType) {
-      setMediaType(filter);
-      setIsMediaLoading(false);
-      setIsMediaLoaded(false);
-    }
-  };
+  const onFilter = useCallback(
+    (filter) => {
+      if (filter !== mediaType) {
+        setMediaType(filter);
+        reloadMedia();
+      }
+    },
+    [mediaType, reloadMedia, setMediaType]
+  );
 
-  const onClose = () => {
-    setIsMediaLoading(false);
-    setIsMediaLoaded(false);
-  };
+  const onClose = useCallback(resetMedia, [resetMedia]);
```

Comment:
I don't think this use of `useCallback` does anything.

The statement says: _"when the value of `resetMedia` changes, assign this changed value to `onClose`, but if it doesn't change, use what it was before."_

Which is completely the same thing as just `const onClose = resetMedia;`.

Reply:
Changed on your recomendation https://github.com/google/web-stories-wp/pull/210#discussion_r375958028 . I didn't see the point, but thought it wouldn't hurt.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/210/commits/c660293bbc9f9e98d78c4cf86a489259a69004b9